README.md

```diff
@@ -72,75 +72,108 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
 ### FancyZones
 
 - Refactored and improved code quality.
+- Fix crashing on moving window between monitors with Win + arrows. (This was a hotfix for 0.67)
+- Fix issue causing window attributes to not be reset properly. (This was a hotfix for 0.67)
+- Fix issue causing window to not be adjusted when layout is changed. (This was a hotfix for 0.67)
+- Fix issue causing window not to be unsnapped on drag started. (This was a hotfix for 0.67)
+- Fix issue causing layouts not to be applied to new virtual desktops. (This was a hotfix for 0.67)
+- Fix issues causing windows not to be restored correctly to their last known zone.
 
 ### File explorer add-ons
 
-- Fixed escaping HTML-sensitive characters when previewing developer files. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Add Developer files previewer option to set max file size and fix styling issue. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
+- Improve Developer files previewer exception handling and printing of error messages.
+- Fix crash when generating PDF and Gcode files thumbnails. (This was a hotfix for 0.67)
 
-### Image Resizer
+### Hosts file editor
 
-- Improved code quality around a silent crash that was being reported to Microsoft servers.
+- Improve hosts file loading. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Improved duplicate hosts finding. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+
+### Keyboard Manager
+
+- Fix typo in Keyboard Manager Editor. Thanks [@ChristianLW](https://github.com/ChristianLW)!
+
+### Mouse Utils
+
+- Resolve grammatical error in Mouse Highlighter description. Thanks [@WordlessSafe1](https://github.com/WordlessSafe1)!
+- New utility: Mouse Jump allows to quickly move the mouse pointer long distances on single or across screens. Thanks [@mikeclayton](https://github.com/mikeclayton)!
+
+### Paste as Plain Text
+
+- New utility: Paste as Plain Text allows pasting the text contents of your clipboard without formatting. Note: the formatted text in the clipboard is replaced with the unformatted text. Thanks [@carlos-zamora](https://github.com/carlos-zamora)!
 
 ### PowerToys Run
 
-- Add option to tab through results only. Thanks [@maws6502](https://github.com/maws6502)!
-- System plugin - Updated Recycle Bin command to allow opening the Recycle Bin. Thanks [@htcfreek](https://github.com/htcfreek)!
-- System plugin - Improved Recycle Bin command to not block PT Run while the deletion is running. Thanks [@htcfreek](https://github.com/htcfreek)!
-- System plugin - Small other changes to improve the usability of the Recycle Bin command. Thanks [@htcfreek](https://github.com/htcfreek)!
-- WindowWalker plugin - Show all open windows with action keyword. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Show Steam (steam://open/) shortcuts in the Program plugin. 
+- Localize paths of Program plugin results. Thanks [@htcfreek](https://github.com/htcfreek)!
```

Comment:
```suggestion
- Localize paths of Program plugin results. Thanks [@htcfreek](https://github.com/htcfreek)!
- Improved stability of localization code. Thanks [@htcfreek](https://github.com/htcfreek)!
```

Reply:
improved stability of the localized code?

Reply:
What I mean I we improved the code to get the localized strings:
- Switched to IShellItem interface.
- Implemented a cache for already localized strings to have a better performance.

This hopefully fixes the problem that it didn't work always in the past.